packages/react/src/Button/Button.module.css

```diff
@@ -113,3 +113,17 @@
 .Button--disabled:not(.Button--secondary) {
   box-shadow: none;
 }
+
+@media (forced-colors: active) {
+  /* Retains focus indicator when in "Windows High Contrast Mode". */
+  .Button:focus:not(.Button--disabled[disabled]) {
+    outline: 3px solid;
+    outline-color: var(--brand-color-focus);
+    outline-offset: 2px;
+  }
+
+  /* Indicates disabled state in WHCM */
+  .Button--label.Button-label--disabled {
+    color: GrayText;
```

Comment:
Question: Are our design tokens not suitable here? Or if `GrayText` is required, should we have a design token that maps to this?

Reply:
I'm not too sure. `GrayText` here only tells WHCM that the text should be styled as if it's disabled. I wonder if we'd benefit from adding this as a token or not? As of right now, I don't think we'll be using it for anything other than color for text that should be disabled. 

Would love to know your thoughts on this.

Reply:
That's good to know. I don't think this needs to be token in that case. It will create unnecessary obfuscation and also isn't something we'd want users to customise from sounds of it.